cdk/lib/prism-access.ts

```diff
@@ -3,20 +3,32 @@ import type { App } from "@aws-cdk/core";
 import { CfnOutput } from "@aws-cdk/core";
 import type { GuStackProps } from "@guardian/cdk/lib/constructs/core";
 import { GuArnParameter, GuStack } from "@guardian/cdk/lib/constructs/core";
+import { AppIdentity } from "@guardian/cdk/lib/constructs/core/identity";
 import { GuRole } from "@guardian/cdk/lib/constructs/iam";
 
 export class PrismAccess extends GuStack {
+  private static app: AppIdentity = {
+    app: "prism",
+  };
+
   constructor(scope: App, id: string, props: GuStackProps) {
     super(scope, id, props);
 
+    /*
+    Looks like some @guardian/cdk constructs are not applying the App tag.
+    I suspect since https://github.com/guardian/cdk/pull/326.
+    Until that is fixed, we can safely, manually apply it to all constructs in tree from `this` as it's a single app stack.
+    TODO: remove this once @guardian/cdk has been fixed.
+     */
+    AppIdentity.taggedConstruct(PrismAccess.app, this);
```

Comment:
This can be seen in the snapshot diff on the [first commit](https://github.com/guardian/prism/pull/155/commits/dc69fbee724965a49d9db54ef356fb18867e368c).